src/polygonParts/models/interfaces.ts

```diff
@@ -18,9 +22,19 @@ export interface InsertPartData extends Readonly<Omit<CommonPayload, 'countries'
  * Polygon parts ingestion payload
  */
 export interface PolygonPartsPayload extends Omit<PolygonPartsPayloadType, 'productType'> {
-  readonly productType: ProductType;
+  readonly productType: RasterProductTypes;
 }
 
+/**
+ * Polygon parts ingestion response
+ */
+export interface CreatePolygonPartsResponse extends PolygonPartsResponse {}
```

Comment:
IMO we can use `PolygonPartsResponse` for both of them

Reply:
ok